### PR TITLE
Tickets/dm 25179: add circular actuators

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/simactuators/circular_tracking_actuator.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    -----
+    Attributes:
+
+    * ``target``: target set by `set_target` (a `path.PathSegment`).
+    * ``path``: actual actuator path (a `path.Path`).
+
+    `set_target` wraps ``target.position`` into the range [0, 360).
+    That is the only guarantee about the wrap of of angles
+    in ``target`` and ``path``. Thus the positions in ``path``
+    can easily be out of that range, as can the compute ``path`` position
```
Comment (wvreeven) [inline python/lsst/ts/simactuators/circular_tracking_actuator.py]: Perhaps my English is failing me, but shouldn't that be "the computed ``path`` position"?

File: python/lsst/ts/simactuators/circular_tracking_actuator.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        Parameters
+        ----------
+        tai : `float`
+            TAI time (unix seconds, e.g. from lsst.ts.salobj.current_tai()).
+        position : `float`
+            Position (deg)
+        velocity : `float`
+            Velocity (deg/sec)
+        direction : `Direction`
+            Desired overall direction of motion.
```
Comment (wvreeven) [inline python/lsst/ts/simactuators/circular_tracking_actuator.py:167]: There is a velocity and a direction, so shouldn't velocity be speed instead?

**Review by wvreeven — APPROVED** [review]
Looks good to me.

File: python/lsst/ts/simactuators/circular_tracking_actuator.py (hunk 2 above)
Comment (r-owen) [reply python/lsst/ts/simactuators/circular_tracking_actuator.py:167]: That is a good question, and the answer is that they are entirely different things. `direction` tells the actuator which way slew to acquire the target, wheres the target velocity tells the actuator what to do once it arrives. Slewing vs. tracking.

I will try to clarify the doc string!

**Pull request merged**